### PR TITLE
Issue 6117: Transactions get aborted due to lease expiry on failure or slowness of pingTxn API

### LINKED ISSUE #6117: Transactions get aborted due to lease expiry on failure or slowness of pingTxn API
**Describe the bug**
There are 3 longevity plans running over this cluster namely medium-scale, medium-scale-tx-bytes & new-scale-tx (custom plan). Their workload is defined: -

medium-scale plan:  ~3 MB/s throughput with 4 writers and 8 readers, scaling based on event ingestion rate at target rate of 200 events/sec
medium-scale-tx-bytes plan:  ~90 KB/s throughput transactional workload (10 events/commit) with 4 writers and 8 readers, scaling based on data ingestion rate at target rate of 10 KB/sec
new-scale-tx plan:  ~10 Mb/s throughput transactional workload (100 events/commit) with 4 writers and 8 readers, scaling based on event ingestion rate at target rate of 50 events/sec
There is a writer failure (Writers: 3/4) for medium-scale-tx-bytes plan on encountering "io.pravega.client.stream.TxnFailedException: Commit transaction failed" on this stamp "2021-01-30 19:22:49". And the particular transaction (txnId=00000f5c-0000-0000-0000-0000006ff1b9) that failed from committing here actually got aborted as pings failed for this txn from renewing the timeout period thereby leading to automatically aborting this txn.

**Additional information**
To address this issue we need a config change

### PR DESCRIPTION
**Pull request by shshashwat**
**Change log description**  
Default timeout at controller config.java increased to 300s from 120s and pingtxn increased to 270s from 90s to recover from the failure due to slowness or failed pingTxn API

**Purpose of the change**  
Fixes #6117

**What the code does**  
Increases the default timeout of transaction from 2 to 5 minutes and pingtxn from 90S to 270s

### CODE REVIEW

File: controller/src/main/java/io/pravega/controller/util/Config.java [hunk 1]
```diff
@@ -174,7 +174,7 @@
             "transaction.lease.count.min", 10000, "transaction.minLeaseValue");
 
     public static final Property<Long> PROPERTY_TXN_MAX_LEASE = Property.named(
-            "transaction.lease.count.max", 120000L, "transaction.maxLeaseValue");
+            "transaction.lease.count.max", 300000L, "transaction.maxLeaseValue");
```
Comment (shiveshr) [inline controller/src/main/java/io/pravega/controller/util/Config.java:177]: maybe increase it to 10 minutes.

File: client/src/main/java/io/pravega/client/stream/EventWriterConfig.java [hunk 2]
```diff
@@ -105,7 +105,7 @@
         private int maxBackoffMillis = 20000;
         private int retryAttempts = 10;
         private int backoffMultiple = 10;
-        private long transactionTimeoutTime = 90 * 1000 - 1;
+        private long transactionTimeoutTime = 270 * 1000 - 1;
```
Comment (shiveshr) [inline client/src/main/java/io/pravega/client/stream/EventWriterConfig.java:108]: this can be as high as max lease value - 1.

**Review by pbelgundi — COMMENTED** [review]
LGTM, lets make sure all tests (including System tests) pass.

**Review by shiveshr — APPROVED** [review]

**Review by pbelgundi — APPROVED** [review]

**Comment by pbelgundi** [comment]
@shshashwat Please update this PR once System tests are successful.

**Comment by shshashwat** [comment]
> @shshashwat Please update this PR once System tests are successful.

The tests are successful.

**Pull request merged**